hs_core/management/commands/update_data_services.py

```diff
@@ -0,0 +1,52 @@
+"""
+This command will update all GeoServer data services.
+
+This will loop through all public composite resources
+containing geographic feature, geographic raster, and/or
+time series content and send an update signal to the Web
+Services Manager application. Web services must be enabled
+in local_settings.py.
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from django.conf import settings
+from hs_core.tasks import update_web_services
+
+
+class Command(BaseCommand):
+    help = "Updates web services for all public resources."
+
+    def add_arguments(self, parser):
+        # A list of resource ID's, or None to check all resources.
+        parser.add_argument("resource_ids", nargs="*", type=str)
+
+    def handle(self, *args, **options):
+        # Run only if web services are enabled.
+        if settings.HSWS_ACTIVATED:
```

Comment:
I think you might find it helpful to spit out the status of this setting.  Right now if this is false the command will just end with zero feedback.

Reply:
print functions revealing the current state of the command (how many resources it will process, how many it has processed so far, how many resources meet the criteria) will be helpful throughout this command.  I imagine it takes awhile to run on prod data so you're going to want it.

Reply:
I added some basic print functions. It’s difficult to get more meaningful updates since the update process is happening on an external service via a bunch of celery tasks.

Reply:
If you use a logger, celery will log out these statements